src/test/java/org/opentripplanner/model/plan/TestItineraryBuilder.java

```diff
@@ -154,6 +164,63 @@ public TestItineraryBuilder bus(int tripId, int startTime, int endTime, Place to
     return bus(tripId, startTime, endTime, TRIP_FROM_STOP_INDEX, TRIP_TO_STOP_INDEX, to, null);
   }
 
+  public TestItineraryBuilder flex(int start, int end, Place to) {
+    if (lastPlace == null) {
```

Comment:
@hannesj

I'm going to leave this code in here even though it's not used right now. Took me a while to figure everything out and it will become useful at some stage.